modules/core/src/opencl/runtime/generator/parser_clamdblas.py

```diff
@@ -94,15 +94,15 @@
 numEnabled = readFunctionFilter(fns, filterFileName)
 
 functionsFilter = generateFilterNames(fns)
-filter_file = open(filterFileName, 'wb')
+filter_file = open(filterFileName, 'w')
```

Comment:
What kind of problem is observed with 'wb'?

Reply:
With Python 3, `'wb'` gives `TypeError: a bytes-like object is required` when we try to write a string to the file.

`'w'` works with both Python 3 and Python 2 strings, and (in our use case) it produces an ASCII-encoded text file.

`'wb'` is (in my view) simply wrong here because it implies that we are writing a binary file. We are not; we are writing a text file.  Despite being conceptually wrong, `'wb'` may work for writing text files in Python 2 because Python 2 represents strings as byte arrays.